xc/xc7/yosys/synth.tcl

```diff
@@ -14,9 +14,14 @@ source [file join [file normalize [info script]] .. utils.tcl]
 # -flatten is used to ensure that the output eblif has only one module.
 # Some of symbiflow expects eblifs with only one module.
 #
+# To solve the carry chain congestion at the output, the synthesis step
```

Comment:
Can you log a bug about this issue?

Reply:
Sure, as soon as I can get a small and reproducible test I'll open an issue